CHANGELOG.md

```diff
@@ -91,66 +92,81 @@ any extra access details
 - Added Concourse CI pipeline
 - Added integration testing for async services
 
-## [3.2.1] - 2017-03-31
+## [3.1.2] - 2017-02-12
 
-- fixed a bug where Spanner instances could be deleted from Google but not deprovisioned in CF
-- fixed a bug where Cloud SQL instances were not being marked as deleted in the Service Broker database
+- fixed a bug where supplying a custom name for Bigtable would cause an error
 
-## [3.3.0] - 2017-03-31
+## [3.1.1] - 2017-02-09
 
-- Added Stackdriver Debugger and Trace support
+- fixed a bug in reading in custom plans
 
-## [3.3.1] - 2017-05-11
+## [3.1.0] - 2017-02-07
 
-- Security updates to address CVE-2017-4975
-- database name is configurable by setting `DB_NAME`
+- Updated vendored packages so that custom UserAgent string gets propagated for storage provision calls
+- Added Bigtable as a service
 
-## [3.3.2] - 2017-05-11
+## [3.0.1] - 2017-01-23
 
-- Security updates to address CVE-2017-4975
-- properly set buildpack
+- Updated default user agent string so that bogus data will not be collected during testing
+- Updated service account bindings to include ProjectId
 
-## [3.3.2] - 2017-05-11
+## [3.0.0] - 2017-01-12
 
-- Security updates to address CVE-2017-4975
-- properly set buildpack
+- Updated pubsub library so that User Agent string gets propagated correctly
+- Updated dependency management system
+- Changed org to system (Broker will need to be uninstalled and reinstalled for this change to take effect)
 
-## [3.4.0] - 2017-05-16
+## [2.1.3] - 2017-01-05
 
-- Add Stackdriver services to tile.yml
+### Fixed
+- fixes a bug where bind calls to ml-api service instances were failing because these service instances don't save
+any extra access details
 
-## [3.4.1] - 2017-05-22
+## [2.1.2] - 2016-12-21
 
-- fixes uninitialized security group
+### Fixed
+- fixes a bug where anything that triggered an install repeat after installing version 2.1.0 or 2.1.1 would cause the
+installation to fail.
 
-## [3.5.1] - 2017-09-06
+## [2.1.1] - 2016-12-02
 
-- added PostgreSQL support to CloudSQL (and migrated existing plans)
-- added Datastore support
+- added new uri parameter to Cloud SQL bind credentials
 
+## [2.1.0] - 2016-12-02
 
-## [3.5.2] - 2017-10-17
+- Remove need for service name for PubSub (topic_name), BigQuery (name), Cloud Storage (name), and Cloud SQL (instance_name)
+- Instance details are now surfaced in bind requests for Pubsub (topic_name and subscription_name),
```

Comment:
If we want to keep our naming consistent with product documentation: s/Pubsub/"Pub/Sub"